Dockerfile

```diff
@@ -1094,7 +1094,7 @@ function create_www_user() {
        --lastname "${_AIRFLOW_WWW_USER_LASTNAME="Admin"}" \
        --email "${_AIRFLOW_WWW_USER_EMAIL="airflowadmin@example.com"}" \
        --role "${_AIRFLOW_WWW_USER_ROLE="Admin"}" \
-       --password "${local_password}" || true
+       --password "${local_password}" > /dev/null 2>&1 || true
```

Comment:
Why this ? At the vvery leat It would be good to add some indication (echo line before) that we are doing it, but the worst part is that if it will fail we will not see error